shared/src/main/scala/effekt/symbols/Name.scala

```diff
@@ -107,11 +109,13 @@ object Name {
    */
   def apply(name: String): Name = {
     assert(name.nonEmpty, "Name cannot be empty.")
+    println(s"Convert name $name")
```

Comment:
You probably want to remove those, right?

Reply:
Yes, just some good old println-debugging.